code/modules/flufftext/Hallucination.dm

```diff
@@ -185,12 +187,16 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 		next_expand = world.time + FAKE_FLOOD_EXPAND_TIME
 
 /datum/hallucination/fake_flood/proc/Expand()
+	for(var/image/I in flood_images)
+		I.alpha = min(I.alpha + 50, 255)
 	for(var/turf/FT in flood_turfs)
 		for(var/dir in GLOB.cardinals)
 			var/turf/T = get_step(FT, dir)
 			if((T in flood_turfs) || !FT.CanAtmosPass(T))
 				continue
-			flood_images += image(image_icon,T,image_state,MOB_LAYER)
+			var/image/new_plasma = image(image_icon,center,image_state,FLY_LAYER)
```

Comment:
T

Reply:
also did you test this? i tried the same thing earlier and it didn't work since you're putting on something that's on a plane below mobs.